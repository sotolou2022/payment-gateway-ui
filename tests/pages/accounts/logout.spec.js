import { mount, enableAutoUnmount } from "@vue/test-utils";
import { beforeEach, afterEach, expect, jest } from "@jest/globals";
import Component from "~~/accounts/logout";
import { createTestingPinia } from "@pinia/testing";

let wrapper;

beforeEach(() => {
  wrapper = mount(Component, {
    global: {
      plugins: [createTestingPinia],
      mocks: {
        useRouter: jest.fn(() => ({
          push: jest.fn(),
        })),
      },
    },
  });
});

enableAutoUnmount(afterEach);

describe("Test suite for accounts logout page", () => {
  it("is a vue component", () => expect(wrapper.vm).toBeTruthy());

  it("match snapshot", () => expect(wrapper.html()).toMatchSnapshot());
});
