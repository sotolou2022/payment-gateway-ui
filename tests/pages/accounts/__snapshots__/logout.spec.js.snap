// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for accounts logout page match snapshot 1`] = `
"<div class=\\"col text-center\\">
  <div class=\\"spinner-border\\" role=\\"status\\"><span class=\\"visually-hidden\\">Loading...</span></div>
  <div class=\\"lead\\">Please wait...</div>
</div>"
`;
